target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mf289f.dts

```diff
@@ -0,0 +1,436 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2022, Pawel Dembicki <paweldembicki@gmail.com>.
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+        model = "ZTE MF289F";
+        compatible = "zte,mf289f";
+
+        aliases {
+                led-boot = &led_internal;
+                led-failsafe = &led_internal;
+                led-running = &led_internal;
+                led-upgrade = &led_internal;
+        };
+
+        chosen {
+                /*
+                 * bootargs forced by u-boot bootipq command:
+                 * 'ubi.mtd=rootfs root=mtd:ubi_rootfs rootfstype=squashfs rootwait'
+                 */
+                bootargs-append = " root=/dev/ubiblock0_1";
+        };
+
+        gpio-restart {
+                compatible = "gpio-restart";
+                gpios = <&tlmm 8 GPIO_ACTIVE_HIGH>;

```

Comment:
Is reset broken without it?
CPU itself has a reset peripheral that powers down the PMIC for a moment and resets the CPU.

Reply:
No it works and is called when you issue "reboot" command.

So can I leave it?

Reply:
@robimarko this is to reset the modem when rebooting the device - it's what stock firmware does, otherwise modem misbehaves after restart - a known quirk of MF286* series.

Reply:
Well, that is a different thing then, please add a comment why its required.

Reply:
comment added as requested